packages/blockchain-wallet-v4-frontend/src/modals/OrderMyCard/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react'
+import { connect, ConnectedProps } from 'react-redux'
+import { compose } from 'redux'
+
+import Flyout, { duration } from 'components/Flyout'
+import { ModalName } from 'data/modals/types'
+import modalEnhancer from 'providers/ModalEnhancer'
+
+import { ModalPropsType } from '../types'
+import CreateCardStep from './CreateCardStep'
+
+const OrderMyCard = (props: Props) => {
+  const [show, setShow] = useState(false)
+
+  useEffect(() => {
+    setShow(true)
+  }, [])
```

Comment:
maybe set show to be true by default, since this will be called once on component render

Reply:
good catch